### ISSUE TITLE
Add 60 more checks

### ISSUE THREAD

**Issue by szepeviktor** [chesio/bc-security#72]
Hello @chesio !

You could steal 30 checks from this HTTP analyzer https://github.com/szepeviktor/wordpress-fail2ban/tree/master/block-bad-requests
and 30 more from this **MU plugin** https://github.com/szepeviktor/wordpress-fail2ban/tree/master/mu-plugin

All the best!

**Comment by chesio** [comment]
Hi Viktor,

Thanks for the tips! I'll check both plugins out, but I have to admit I tend to think that such checks are more suited for `.htaccess` file or [auto_prepend_file](https://secure.php.net/manual/en/ini.core.php#ini.auto-prepend-file). Or MU plugin as last resort.

Anyway, I'll have a look what would make sense to me to include in a regular plugin.

**Comment by szepeviktor** [comment]
Yes, sometimes the Bad_Request class is loaded in `auto_prepend_file`.

**Comment by szepeviktor** [comment]
My WAF got renamed!
See readme: https://github.com/szepeviktor/wordpress-fail2ban

WAF for WordPress now consists of Http_Analyzer and Core_Events
